### PR TITLE
Change jexl integration test script.

### PR DESCRIPTION
**Pull request by jrbenny35**
This is now causing some failures but it matches a bit better to what Firefox does.

### CODE REVIEW

**Review by jaredlockhart — CHANGES_REQUESTED** [review]
Oh this misses the other thing I mentioned about reordering the generation of the targeting strings here: https://github.com/mozilla/experimenter/issues/6475#issuecomment-928055547

File: app/experimenter/experiments/models/nimbus.py [hunk 1]
```diff
@@ -323,7 +323,7 @@ def proposed_end_date(self):
     def computed_enrollment_days(self):
         paused_change = (
             self.changes.all()
-            .filter(NimbusChangeLog.Filters.IS_APPROVED_PAUSE)
```
Comment (jaredlockhart) [inline app/experimenter/experiments/models/nimbus.py:326]: Oh that all snuck in from another PR, just revert those lines

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 2]
```diff
@@ -542,34 +539,6 @@ def test_computed_enrollment_days_returns_fallback(self):
...
-            new_status=NimbusExperiment.Status.LIVE,
-        ).update(changed_on=datetime.datetime.now() - datetime.timedelta(days=5))
-        experiment.changes.filter(experiment_data__is_paused=True).update(
-            changed_on=datetime.datetime.now()
-        )
-        self.assertEqual(
-            experiment.computed_enrollment_days,
-            expected_days,
-        )
-
```
Comment (jaredlockhart) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:572]: And all this stuff

Comment (jrbenny35) [reply app/experimenter/experiments/tests/test_models/test_models_nimbus.py:572]: Where the heck did this come from

Comment (jaredlockhart) [reply app/experimenter/experiments/tests/test_models/test_models_nimbus.py:572]: Just landed in main after I closed my PR

Comment (jaredlockhart) [reply app/experimenter/experiments/tests/test_models/test_models_nimbus.py:572]: If you're using VSCode it has a thing where you can revert individual blocks of changes within files

Comment (jrbenny35) [reply app/experimenter/experiments/tests/test_models/test_models_nimbus.py:572]: Oh I get it.

File: app/experimenter/experiments/tests/test_models/test_models_nimbus.py [hunk 3]
```diff
@@ -490,25 +490,22 @@ def test_should_end_enrollment_returns_True_after_proposed_enrollment_end_date(s
...
 
         experiment.changes.filter(experiment_data__is_paused=True).update(
             changed_on=datetime.datetime.now()
         )
 
         self.assertEqual(
             experiment.computed_enrollment_days,
-            expected_days,
+            3,
         )
```
Comment (jaredlockhart) [inline app/experimenter/experiments/tests/test_models/test_models_nimbus.py:509]: This needs to get reverted too

**Comment by jaredlockhart** [comment]
hey @piatra  in the ExperimentManager jsm, there's this bit: https://searchfox.org/mozilla-central/source/toolkit/components/nimbus/lib/ExperimentManager.jsm#77-82

but when i try it locally like this

https://github.com/mozilla/experimenter/blob/62136e96ab1b6d64d3cf636c549c981df01569e7/app/tests/integration/nimbus/utils/filter_expression.js#L25-L32

i get 

Screen Shot 2021-09-30 at 4 32 13 PM

which is used in the first run targeting jexls like this:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants/nimbus.py#L143

so I think this is causing all the first run targeting tests to fail atm.  

Is there something we're not setting up correctly in the test?

**Comment by piatra** [comment]
yeah so you're doing this:
```js
const experimentManager = new ExperimentManager._ExperimentManager() 
```
This creates a new uninitialized experiment manager. `activeExperiments` will never resolve because it needs the experiment store and a whole bunch of setup to be useful.
You need a reference to the actual `ExperimentManager` in use (that should be `ExperimentManager.ExperimentManager` based on how the imports look/work in that script)
Alternatively maybe give another try to using the `evaluateJexl` method now that I've changed the return type (not sure if that was the only problem preventing this option)

**Comment by jaredlockhart** [comment]
Aha! Thank you @piatra let me try your new modified `evaluateJexl`

**Comment by jaredlockhart** [comment]
AWESOME this works!!

```js
-    const experimentManager = new ExperimentManager._ExperimentManager()
-
     const context = TargetingContext.TargetingContext.combineContexts(
         _experiment,
-        experimentManager.createTargetingContext(),
+        ExperimentManager.ExperimentManager.createTargetingContext(),
```

**Review by jaredlockhart — APPROVED** [review]
YESSSSSS 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Pull request merged**